docs/topics/OAuth2.md

```diff
@@ -73,7 +73,7 @@ When someone navigates to this URL, they will be prompted to authorize your appl
 ###### Redirect URL Example
 
 ```
-https://nicememe.website/?code=NhhvTDYsFcdgNLnnLijcl7Ku7bEEeee&state=15773059ghq9183habn
+https://redirect.com/?code=NhhvTDYsFcdgNLnnLijcl7Ku7bEEeee&state=15773059ghq9183habn
```

Comment:
I would revert this as "redirect.com" is a real service that would essentially get free advertisement by using their domain. If you wanted it to be more professional, then you should use an IANA-managed reserved domain such as example.com.

Reply:
oh ok

Reply:
I agree with @BrightSkyz. Is there a reason why you want to change the domains? I think everyone knows those are fake URIs, and if not, they will know if they look them up.

Reply:
nicememe.website made firefox give me a `Warning: Potential Security Risk Ahead` message. About that...

Reply:
@Elektron-blip Most likely an expired SSL cert.

Reply:
But I think [example.com](https://example.com/) would be best

Reply:
> @Elektron-blip Most likely an expired SSL cert.

Ah I see.